tests/run-make/print-to-output/rmake.rs

```diff
@@ -0,0 +1,66 @@
+//! This checks the output of some `--print` options when
+//! output to a file (instead of stdout)
+
+extern crate run_make_support;
+
+use std::ffi::OsString;
+
+use run_make_support::{rustc, target, tmp_dir};
+
+fn main() {
+    // Printed from CodegenBackend trait impl in rustc_codegen_llvm/src/lib.rs
+    check(
+        /*target*/ &target(),
+        /*option*/ "relocation-models",
+        /*includes*/ &["dynamic-no-pic"],
+    );
+
+    // Printed by compiler/rustc_codegen_llvm/src/llvm_util.rs
+    check(
+        /*target*/ "wasm32-unknown-unknown",
+        /*option*/ "target-features",
+        /*includes*/ &["reference-types"],
+    );
+
+    // Printed by C++ code in rustc_llvm/llvm-wrapper/PassWrapper.cpp
+    check(
+        /*target*/ "wasm32-unknown-unknown",
+        /*option*/ "target-cpus",
+        /*includes*/ &["generic"],
+    );
+}
+
+fn check(target: &str, option: &str, includes: &[&str]) {
+    fn _inner(output: &str, includes: &[&str]) {
+        for i in includes {
+            assert!(output.contains(i), "output doesn't contains: {}", i);
+        }
+    }
+
+    let stdout: String;
+    let output: String;
```

Comment:
This can become

```rs
let stdout = {
    // ...
    _inner(&stdout, includes);
    stdout
};

let output = {
    // ...
    _inner(&output, includes);
    output
};
```

right? Just a stylistic nit.